src/components/screens/monitor/delegates/forgingDetails.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { Link } from 'react-router-dom';
+import Box from '../../../toolbox/box';
+import BoxHeader from '../../../toolbox/box/header';
+import BoxContent from '../../../toolbox/box/content';
+import styles from './forgingDetails.css';
+import LiskAmount from '../../../shared/liskAmount';
+import { tokenMap, initialLSKSupply } from '../../../../constants/tokens';
+import routes from '../../../../constants/routes';
+
+const ForgingDetails = ({
+  t, delegates, sortDirection, networkStatus,
+}) => {
+  const delegatesList = sortDirection && sortDirection.includes('asc') ? [...delegates.data] : [...delegates.data].reverse();
+  const totalForged = networkStatus && networkStatus.data.supply - initialLSKSupply;
+
+  const lastForger = [...delegatesList].sort(
```

Comment:
💅 I don't see why this extra spread is needed
```suggestion
  const lastForger = delegatesList.sort(
```